plugins/polls/db/migrate/20181110000000_create_polls.rb

```diff
@@ -0,0 +1,37 @@
+class CreatePolls < ActiveRecord::Migration
+  def change
+    create_table :polls do |t|
+      t.integer :created_by_user_id, null: false
+      t.string :name, null: false
+      t.text :description
+      t.datetime :starts
+      t.datetime :ends
+      t.boolean :one_vote_per_ordergroup, default: false, null: false
+      t.text :required_ordergroup_custom_fields
+      t.text :required_user_custom_fields
```

Comment:
I think it would be important to explain the custom fields functionality.

Reply:
done